joomla/joomla-cms
# Bootstrap collapse asset for sub-categories

### Summary of Changes

Content, Contact and Newsfeeds components don't load Bootstrap collapse asset and expanding sub-categories is broken.

We need to load `bootstrap.collapse` asset like it's currently done for `com_search/tmpl/default-form.php`

### Testing Instructions

Use any non-native template, i.e. Gantry.
See child sub-categories in com_content, com_contact and com_newsfeeds category view, click "plus" icon to expand sub-categories.

### Actual result BEFORE applying this Pull Request

Nothing happens.

### Expected result AFTER applying this Pull Request

Sub-categories are expanded.

### Documentation Changes Required

No.
